pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -124,6 +151,13 @@ mod pallet {
         BalanceOf<T>,
     >;
 
+    /// Pair of (last_era_index, current_era_index).
+    ///
+    /// Note: In case of offchain-worker crashed, the last_era_index should also be recorded.
+    #[pallet::storage]
+    #[pallet::getter(fn era_index_pair)]
+    pub type EraIndexPair<T: Config> = StorageValue<_, (Option<EraIndex>, EraIndex), ValueQuery>;
```

Comment:
we should store things in different storage because sometimes we just need one of them, also we will never have `None` case, if stake-client crashes it'll try to reapply eras

Reply:
I thought It doesn't make sense to have a default value for `previous_era` before we first update the era index. But it's okay to do so,  cause we don't read that value before updating. On the other hand, since they're always updated at the same time, It might be better to use a pair to record them.

Reply:
but when we change era we just need `current_era`

Reply:
when we stake/unstake we just need `current_era`

Reply:
Fix it in next commit.

Reply:
I think we should prefer having more meaningful names/functions